src/main/java/com/minecolonies/colony/ColonyManager.java

```diff
@@ -614,7 +696,7 @@ public static void onWorldUnload(@NotNull World world)
     {
         if (!world.isRemote)
         {
-            List<Colony> worldColonies = coloniesByWorld.get(world.provider.getDimensionId());
+            List<Colony> worldColonies = coloniesByWorld.get(world.provider.getDimension());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'worldColonies' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)